rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ClientBase.java

```diff
@@ -76,6 +76,18 @@ public TaskResult getTaskResultFromIncarnation(JobId arg0, String arg1, int arg2
         throw newUnsupportedOperationException();
     }
 
+    @Override
+    public List<TaskResult> getTaskResultAllIncarnations(JobId arg0, String arg1)
+            throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
+        throw newUnsupportedOperationException();
+    }
+
+    @Override
+    public List<TaskResult> getTaskResultAllIncarnations(String arg0, String arg1)
+            throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)